app/assets/javascripts/discourse/lib/keyboard_shortcuts.js

```diff
@@ -208,6 +209,17 @@ Discourse.KeyboardShortcuts = Ember.Object.createWithMixins({
     }
   },
 
+  _stopCallback: function() {
+    const oldStopCallback = this.keyTrapper.stopCallback;
+
+    this.keyTrapper.stopCallback = function(e, element, combo) {
```

Comment:
Per the default configuration from [Mousetrap's documentation](https://craig.is/killing/mice#api.stopCallback), I'm hijacking the callback here to fire the event for search.